boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -82,6 +82,52 @@ config REGULATOR
 
 endif # !TRUSTED_EXECUTION_SECURE
 
+if BOARD_ENABLE_USB_CDC
+
+config USB_DEVICE_PRODUCT
+	default "Thingy:53 Application"
+
+config USB_DEVICE_VID
+	default 0x1915
+
+config USB_DEVICE_PID
+	default 0x530C
+
+config USB_DEVICE_STACK
+	default y
+
+config USB_CDC_ACM
+	default y
+
+config UART_CONSOLE
+	default CONSOLE
+
+config USB_DEVICE_INITIALIZE_AT_BOOT
+	default y if !MCUBOOT
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
+	default SHELL
+
+config USB_DEVICE_REMOTE_WAKEUP
+	default n
+
+if LOG
+
+# Set USB and CDC log level to error only
+config USB_CDC_ACM_LOG_LEVEL
```

Comment:
You have to disable USB_CDC_ACM_LOG_LEVEL otherwise CI would fail for this board.
```
config USB_CDC_ACM_LOG_LEVEL
        default 0
```